test/verify/check-metrics

```diff
@@ -416,8 +416,8 @@ class TestHistoryMetrics(testlib.MachineCase):
         # Now add the journal
         # Journal was recorded on Fedora 33 and when trying to use it with older systemd it fails with:
         # `Journal file /var/log/journal/*/journal.journal uses an unsupported feature, ignoring file.`
```

Comment:
This comment is still relevant but also not relevant anymore as we now skip for a different reason?

Reply:
Right, I updated the comment. While I was at it, I also dropped the RHEL 8 bits, as we don't test that any more.